NOTES.md

```diff
@@ -1,3 +1,191 @@
+# Release 1.16.0 (Stand By Me)
+
+Released on 2015-01-28
+
+The highlights of this release are numerous and exciting. Read the
+[release blog post][1.16-blog] for more details.
+
+[1.16-blog]: http://rethinkdb.com/blog/1.16-release/
+
+There may or may not be API-breaking changes in this release.
+
+## New Features ##
+
+* ReQL admin: a new cluster management and monitoring API (#2169)
+  * Added a system database named `rethinkdb`
+    * The `cluster_config` table contains cluster-wide settings (#2924)
+    * The `current_issues` table lists issues affecting the cluster and suggests solutions (#2864, #3258)
+    * The `db_config` table allows listing and renaming databases (#151, #2871)
+    * The `jobs` table lists running jobs (#3115)
+    * The `logs` table for accessing the server logs (#2884)
+    * The `server_config` table for managing servers (#2873)
+    * The `server_status` table contains information about the servers in the cluster (#2923, #3270)
+    * The `stats` table contains real-time statistics about the server (#2885)
+    * The `table_config` table allows listing, creating, modifying and dropping tables (#2870)
+    * The `table_status` table contains extra information about each table (#2983, #3269)
+    * The hidden `_debug_table_status` table contains internal debugging information (#2901)
```

Comment:
I suggest not listing the hidden tables. I don't want users to start using them like they did with `/ajax`.